aten/src/ATen/native/TensorShape.cpp

```diff
@@ -25,8 +25,24 @@
 #include <algorithm>
 #include <cstdint>
 #include <vector>
+#include <iostream>
 
 namespace at {
+
+namespace meta {
+
+TORCH_META_FUNC(transpose_copy) (const Tensor& self, const int64_t dim0, const int64_t dim1) {
+  auto ndims = self.dim();
+  auto dim0_ = maybe_wrap_dim(dim0, ndims);
+  auto dim1_ = maybe_wrap_dim(dim1, ndims);
+  c10::TensorOptions options = self.options();
+  DimVector sizes(self.sizes().begin(), self.sizes().end());
+  std::swap(sizes[dim0_], sizes[dim1_]);
+  set_output(sizes, options);
```

Comment:
Currently, strides of the resulting tensors would be different when using `transpose` and `transpose_copy`. I think the strides should match for strided tensor inputs.

Reply:
Makes sense. That is, `x.transpose_copy(dim0, dim1)` is defined as `x.transpose(dim0, dim1).clone()` where `clone` preserves strides. Will fix it..

Reply:
Fixed.